Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/cluster/ClusterDetail.java

```diff
@@ -329,4 +333,32 @@ public String getYarnNMConnectionUrl() {
         return URI.create(getConnectionUrl()).resolve("yarnui/ws/v1/cluster/apps/").toString();
     }
 
+    @Override
+    public SparkSubmitStorageType getDefaultStorageType() {
+        return SparkSubmitStorageType.DEFAULT_STORAGE_ACCOUNT;
+    }
+
+    @Override
+    public SparkSubmitStorageTypeOptionsForCluster getStorageOptionsType() {
+        StorageAccountTypeEnum type = StorageAccountTypeEnum.UNKNOWN;
+        try {
+            type = getStorageAccount().getAccountType();
+        } catch (HDIException e) {
+            try {
+                getConfigurationInfo();
+                type = getStorageAccount().getAccountType();
+            } catch (IOException | HDIException | AzureCmdException ex) {
```

Comment:
If the exception is ignored


```suggestion
            } catch (IOException | HDIException | AzureCmdException ignored) {
```